lib/rbac/roles.rb

```diff
@@ -39,6 +39,15 @@ def delete(role)
       end
     end
 
+    def self.assigned_role?(role_name)
+      opts = { :limit => 100,
+               :scope => 'principal' }
```

Comment:
@lindgrenj6 should we also set the 
```
  :name => role_name
```
which will limit it to the role you are interested in otherwise it will fetch extra roles over the network which is not needed

Reply:
Like we do here
https://github.com/ManageIQ/catalog-api/blob/7ae8422696a4cdec369c9e4746a7683f61d4ec6f/lib/rbac/roles.rb#L46